src/reporting/ReportingWindow.sol

```diff
@@ -24,10 +25,10 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable, IReportingWi
     Set private markets;
     Set private limitedReporterMarkets;
     Set private allReporterMarkets;
+    uint256 private totalMarketsThatRequiredReporting;
     mapping(address => Set) private reportsByReporter;
+    mapping(address => bool) private reporterDoneReporting;
```

Comment:
Consider having a single `mapping(address => ReportingStatus) reportingStatii` (come up with better name) where
```
struct ReportingStatus {
    Set marketsReportedOn;
    bool finishedReporting;
}
```
I'm curious if this would help make some of the other code for reporter tracking more readable (last I checked a while ago, the code was difficult to follow).

Reply:
Will try this!